api.planx.uk/modules/gis/service/digitalLand.ts

```diff
@@ -71,15 +71,30 @@ async function go(
   geom: string,
   extras: Record<string, string>,
 ): Promise<GISResponse> {
-  // generate list of digital land datasets we should query based on 'active' planx schema variables
+  // generate list of digital land datasets we should query and their associated passport values
   const activeDatasets: string[] = [];
-  Object.keys(baseSchema).forEach((key) => {
-    if (baseSchema[key]["active"]) {
-      baseSchema[key]["digital-land-datasets"]?.forEach((dataset: string) => {
-        activeDatasets.push(dataset);
-      });
-    }
-  });
+  let activeDataValues: string[] = [];
+  if (extras?.vals) {
```

Comment:
It's tricky as these aren't 1:1 Planning.data datasets, but I think we should use a more meaningful param name here - maybe `datasets`, but this isn't ideal given the context here and the existing meaning `datasets` has.

We also need to add this new param to Swagger docs 👍

Reply:
Went with `vals` because the response includes a `val` property for each item - agree not most semantically meaningful though ! Continue to strongly oppose "datasets" though as that already corresponds to Planning Data _source_, not our curated constraint item which rely on one or many source datasets.